recipes/larch-phylo/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "larch" %}
+{% set package_name = "larch-phylo" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ package_name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/matsengrp/larch.git
+  git_rev: conda-forge
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - yasm
+    - automake
+    - autoconf
+
+  host:
+    - libtool
+    - zlib
+    - openmpi ==5.0.8
+    - openmpi-mpicc ==5.0.8
+    - openmpi-mpicxx ==5.0.8
```

Comment:
Please read our channel documentation about how to build a package which depends on MPI.

https://conda-forge.org/docs/maintainer/knowledge_base/#message-passing-interface-mpi

The mpicc,mpicxx,mpifortran packages are compiler packages which are not appropriate for the host environment. They are also just convenience packages for end users, not for use in recipes.